ideascube/conf/idb_grc_eleonas.py

```diff
@@ -26,25 +26,6 @@
         'id': 'mediacenter',
     },
     {
-        'id': 'wikipedia',
```

Comment:
@fheslouin said me on IRC that those contents not installed by package use the ".old" template.
But yes, it's better to synchronized with ansiblecube team before the merge.